Decred Wallet/Features/Security/PasswordSetupViewController.swift

```diff
@@ -78,12 +78,12 @@ class PasswordSetupViewController: SecurityBaseViewController, UITextFieldDelega
     func validatePasswordsAndProceed() -> Bool {
         let password = self.tfPassword.text ?? ""
         if password.length == 0 {
-            self.showMessageDialog(title: "Error", message: "Empty password not allowed")
+            self.showMessageDialog(title: LocalizedStrings.error, message: LocalizedStrings.emptyPasswordNotAllowed)
             return false
         }
         
         if self.tfPassword.text != self.tfConfirmPassword.text {
-            self.showMessageDialog(title: "Error", message: "Passwords do not match")
+            self.showMessageDialog(title: LocalizedStrings.error, message: LocalizedStrings.passwordsDoNotMatch)
```

Comment:
Don't see a need for 2 "Passwords do not match" translations: `LocalizedStrings.passwordsDoNotMatch` and `LocalizedStrings.passwordDoNotMatch` (used in line 61 of this file). If the usage in line 61 MUST be capital (I don't think so), but if it's compulsory, we can convert the non-capital translation to capital. Retain `LocalizedStrings.passwordDoNotMatch`, delete `LocalizedStrings.passwordDoNotMatch`.

Reply:
This will be sorted out in net PR Translation